code/hello/README.md

```diff
@@ -3,12 +3,18 @@
 
 This example should help to check that your machine is well installed.
 
-## g++ & make
+## make & cmake
 
-Try:
+On linux, provided you have a "recent enough" g++, this should work directly:
+```
+make
+./hello
+```
+
+On windows, it may be mandatory to run `cmake` first:
 ```
+cmake .
 make
-export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:.
 ./hello
```

Comment:
I am confused by this new comment. The `Makefile` on master works out of the box in WSL. On native Windows, you would run:
```
mkdir build
cd build
cmake ..
cmake --build .
Debug/hello.exe
```

Reply:
Ok. I am fully cmake allergic, thus the bad comments. I will  insert you above rule, which is fine with me, since it does not overwrite the current Makefile, as I did when typing `cmake . ` . As I said in some other issue, I am not at ease with this double "make" and/or "cmake" parallel configurations.

Reply:
And then a `build` in `.gitignore` is enough for all the cmake by-products.